packages/lib/server/repository/user.ts

```diff
@@ -337,6 +337,22 @@ export class UserRepository {
     };
   }
 
+  static enrichUserWithItsProfileBuiltFromUser<T extends { id: number; username: string | null }>({
```

Comment:
A non async method that just builds the profile like object from User table instead of querying profile table.